posts/2022-03-15-ga-release-post.adoc

```diff
@@ -0,0 +1,155 @@
+seo-description: Open Liberty 22.0.0.3 allows SQL Operations to be retried in transaction recovery logs. This release also includes several significant bug fixes. In addition, we have published a brand new Open Liberty guide on containerizing microservices with Podman!
+blog_description: Open Liberty 22.0.0.3 allows SQL Operations to be retried in transaction recovery logs. This release also includes several significant bug fixes. In addition, we have published a brand new Open Liberty guide on containerizing microservices with Podman!
+
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Retrying SQL operations in transaction recovery logs and more in Open Liberty 22.0.0.3
+Ryan Storey <https://github.com/ryan-storey>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Open Liberty 22.0.0.3 allows SQL Operations to be retried in transaction recovery logs. This release also includes several significant bug fixes. In addition, we have published a brand new Open Liberty guide on containerizing microservices with Podman!
+In link:{url-about}[Open Liberty] 22.0.0.3:
+
+* <<sql, Allow SQL Operations to be retried in transaction recovery logs>>
+
+Along with the new features and functions added to the runtime, we’ve also made <<guides, updates to our guides>>.
+
+View the list of fixed bugs in link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A22003+label%3A%22release+bug%22+[22.0.0.3].
+
+[#run]
+== Run your apps using 22.0.0.3
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.openliberty</groupId>
+    <artifactId>openliberty-runtime</artifactId>
+    <version>22.0.0.3</version>
+    <type>zip</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version: '[22.0.0.3,)'
+}
+----
+
+Or if you're using Docker:
+
+[source]
+----
+FROM open-liberty
+----
+
+Or take a look at our link:{url-prefix}/downloads/[Downloads page].
+
+[link=https://stackoverflow.com/tags/open-liberty]
+image::img/blog/blog_btn_stack.svg[Ask a question on Stack Overflow, align="center"]
+
+[#sql]
+=== Allow SQL Operations to be retried in transaction recovery logs
+
+In Liberty, transaction services are implicitly activated when you specify features that require the use of transactions; for example, `jpa-2.0`, `jdbc-4.0`, and `wasJmsServer-1.0`.
+
+When Liberty detects a failure in writing to its transaction recovery logs, it will invalidate the logs and disallow further global transactions. In order to do further transactional work, the customer must restart the server. This means applications have to be reloaded when the server restarts. This can take a considerable amount of time for some applications.
```

Comment:
suggested edit:

When Liberty detects a failure to write to its transaction recovery logs, it invalidates the logs and disallows further global transactions. To resume transactional work, you have to restart the server and reload any applications, which can take considerable time in some cases.